internal/extract.go

```diff
@@ -78,6 +83,10 @@ var _ io.Writer = &DevNullWriter{}
 func extractOneTar(tarInterpreter TarInterpreter, source io.Reader) error {
 	tarReader := tar.NewReader(source)
 
+	if groupInterpreter, ok := tarInterpreter.(GroupTarInterpreter); ok {
```

Comment:
Little abstraction leak here, but I think that's more appropriate than duplication of all methods for GroupTarInterpreter